sdk/quota/Azure.ResourceManager.Quota/src/Custom/ArmQuotaModelFactory.cs

```diff
@@ -0,0 +1,80 @@
+        /// <param name="allocatedToSubscriptionsValue"> Quota allocated to subscriptions. </param>
+        /// <returns> A new <see cref="Models.GroupQuotaDetails"/> instance for mocking. </returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static GroupQuotaDetails GroupQuotaDetails(string resourceName, long? limit, string comment, string unit, long? availableLimit = null, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Quota.Models.SubscriptionAllocatedQuota> allocatedToSubscriptionsValue = null, string value = null, string localizedValue = null)
+        {
+            return GroupQuotaDetails(
+                resourceName,
+                limit,
+                comment,
+                unit,
+                value,
+                localizedValue,
+                availableLimit,
+                allocatedToSubscriptionsValue);
+        }
+
+        /// <summary> Initializes a new instance of <see cref="Models.GroupQuotaLimitProperties"/>. </summary>
+        /// <param name="resourceName"> The resource name, such as SKU name. </param>
+        /// <param name="limit"> The current Group Quota Limit at the parentId level. </param>
+        /// <param name="comment"> Any comment related to quota request. </param>
+        /// <param name="unit"> The usages units, such as Count and Bytes. When requesting quota, use the **unit** value returned in the GET response in the request body of your PUT operation. </param>
+        /// <param name="value"> Resource name. </param>
+        /// <param name="localizedValue"> Resource display name. </param>
+        /// <param name="availableLimit"> The available Group Quota Limit at the MG level. This Group quota can be allocated to subscription(s). </param>
+        /// <param name="allocatedToSubscriptionsValue"> Quota allocated to subscriptions. </param>
+        /// <returns> A new <see cref="Models.GroupQuotaLimitProperties"/> instance for mocking. </returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static GroupQuotaLimitProperties GroupQuotaLimitProperties(string resourceName, long? limit, string comment, string unit, long? availableLimit = null, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Quota.Models.SubscriptionAllocatedQuota> allocatedToSubscriptionsValue = null, string value = null, string localizedValue = null)
+        {
+            return GroupQuotaLimitProperties(
+                resourceName,
+                limit,
+                comment,
+                unit,
+                value,
+                localizedValue,
+                availableLimit,
+                allocatedToSubscriptionsValue);
+        }
+
+        /// Add this custom code because its properties order changed, but the changed properties are all string, this actually replaces its generated overload.
```

Comment:
```suggestion
        // Add this custom code because its properties order changed, but the changed properties are all string, this actually replaces its generated overload.
```
3 slashes pollute the xml doc - this should only be informational for the purpose of customization code.